deploy/app/example.yaml

```diff
@@ -0,0 +1,13 @@
+address: "127.0.0.1" # address app will run on
+host: "domain.com" # domain/ip of the server for generating links (if deployed in docker host and address may not match)
+# port is configured as APP_PORT env variable
```

Comment:
I find it a bit awkward that part of the configuration is in the file and part in the environment variables.

I'd suggest we move all settings to environment variables, as they are more convenient in Dockerized environments. But this is out of scope for this PR